### PR TITLE
adapter and wire's handles work at the same time

### PR DESCRIPTION
**Pull request by swhgoon**
Tried with code:)
```
var gun = Gun({
  file : "db/db.json",
  level: levelup('db/down.json', {db: require('jsondown')})
});
```

### CODE REVIEW

**Comment by PsychoLlama** [comment]
Hey, thanks for the pull request @swhgoon! I'm finishing up another project atm, but I'll try to review this asap.

File: src/index.js [hunk 1]
```diff
@@ -13,12 +13,18 @@ Gun.on('opt').event((gun, options) => {
...
+	var wireGet = wire.get
+	var wirePut = wire.put
 
 	// Register the driver.
 	gun.opt({
 		wire: {
-			get: wire.get || adapter.read,
-			put: wire.put || adapter.write,
+		      get: function (lex, cb, o) {
+			wireGet(lex, cb, o) || adapter.read(lex, cb)
```
Comment (PsychoLlama) [inline src/index.js]: `wireGet` and `wirePut` aren't always defined, which is why [the build](https://travis-ci.org/PsychoLlama/gun-level/jobs/160968616) failed. If we make sure they're defined before calling them it should fix the errors.
We'll always want to call our own driver (`adapter.read`), but if `wire.get` is defined, then we can call both. The `wire.get` hook will likely refer to a different driver, such as websockets or json.

File: src/index.js [hunk 2]
```diff
@@ -13,12 +13,18 @@ Gun.on('opt').event((gun, options) => {
 	const adapter = Adapter.from(level);
 
 	const { wire } = gun.__.opt;
+	var wireGet = wire.get
```
Comment (PsychoLlama) [inline src/index.js]: Lower priority: for consistency, can we change `var` to `const` and add semicolons? `npm run lint` should help.

**Comment by swhgoon** [comment]
Sorry for the semicolons, I come from `scala` & `scalajs`, with less knowledge about `js`...

**Comment by PsychoLlama** [comment]
No worries! JavaScript is pretty chill whether you use semicolons or not. It's more about keeping things consistent.
Looks like all the issues have been addressed, thanks for the contribution, @swhgoon! You're awesome.

**Comment by PsychoLlama** [comment]
Hey, mind if I add you to the contributors list on the readme?

**Pull request merged**